aiplatform/snippets/embeddings.go

```diff
@@ -28,40 +27,34 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-func embedTexts(
-	project, location string, texts []string) ([][]float32, error) {
+func embedTexts(project, location, model string) ([][]float32, error) {
 	ctx := context.Background()
 
 	apiEndpoint := fmt.Sprintf("%s-aiplatform.googleapis.com:443", location)
-	model := "text-embedding-004"
-	task := "QUESTION_ANSWERING"
-	customOutputDimensionality := 5
+	dimensionality := 5
+	texts := []string{"banana muffins? ", "banana bread? banana muffins?"}
```

Comment:
question: why are we passing in the model as a string but hard-coding the prompt in the sample?

Reply:
Good catch! Just noticed that in those samples used 2 different models in test, which is not straightforward.
I will move it into the sample and leave a comment